docs/data/sdks/sdk-maintenance-and-support.md

```diff
@@ -0,0 +1,88 @@
+| :material-apple-ios: [iOS SDK](./ios-swift/index.md)<br/>:octicons-package-16: `AmplitudeSwift`<br/>[:material-github: Amplitude-Swift](https://github.com/amplitude/Amplitude-Swift)                                                                                                                                                              | 1.x       | Beta           | N/A        |                         |
+| :material-apple-ios: [iOS SDK](./ios/index.md)<br/>:octicons-package-16: [`Amplitude`](https://cocoapods.org/pods/Amplitude-iOS)<br/>[:material-github: Amplitude-iOS](https://github.com/amplitude/Amplitude-iOS)                                                                                                                                 | 1.x - 8.X | GA             | 2014-06-05 |                         |
+| :material-language-java: [Java SDK](./java/index.md)<br/>:octicons-package-16: [`com.amplitude.:java-sdk`](https://mvnrepository.com/artifact/com.amplitude/java-sdk)<br/>[:material-github: Amplitude-Java](https://github.com/amplitude/Amplitude-Java)                                                                                          | 1.x       | GA             | 2021-06-15 |                         |
+| :material-nodejs: [Node SDK](./typescript-node/index.md)<br/>:octicons-package-16: [`@amplitude/analytics-node`](https://www.npmjs.com/package/@amplitude/analytics-node)<br/>[:material-github: Amplitude-Typescript](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-node)                                        | 1.x       | GA             | 2022-12-10 |                         |
+| :material-nodejs: [Node SDK](./node/index.md) <br/>:octicons-package-16: [`@amplitude/node`](https://www.npmjs.com/package/@amplitude/node)<br/>[:material-github: Amplitude-Node](https://github.com/amplitude/Amplitude-Node)                                                                                                                    | 1.x       | Maintenance    | 2020-09-25 |                         |
+| :material-language-python: [Python SDK](./python/index.md)<br/>:octicons-package-16: [`amplitude-analytics`](https://pypi.org/project/amplitude-analytics/)<br/>[:material-github: Amplitude-Python](https://github.com/amplitude/Amplitude-Python)                                                                                                | 1.x       | GA             | 2022-06-29 |                         |
+| :material-react: [React SDK](https://github.com/amplitude/react-amplitude#readme)<br/>:octicons-package-16: [`@amplitude/react-amplitude`](https://www.npmjs.com/package/@amplitude/react-amplitude)<br/>[:material-github: react-amplitude](https://github.com/amplitude/react-amplitude)                                                         | 1.x       | End of support | 2019-05-27 |                         |
```

Comment:
why is it end of support instead of maintenance?

Reply:
This is the `react-amplitude` SDK that is true React, not RN. It has not been maintained since I joined.